system/component_state_monitor/config/topics.yaml

```diff
@@ -0,0 +1,182 @@
+- module: map
+  mode: [online, planning_simulation]
+  type: launch
+  args:
+    node_name_suffix: vector_map
+    topic: /map/vector_map
+    topic_type: autoware_auto_mapping_msgs/msg/HADMapBin
+    best_effort: false
+    transient_local: true
+    warn_rate: 0.0
+    error_rate: 0.0
+    timeout: 0.0
+
+- module: map
+  mode: [online]
+  type: launch
+  args:
+    node_name_suffix: pointcloud_map
+    topic: /map/pointcloud_map
+    topic_type: sensor_msgs/msg/PointCloud2
+    best_effort: false
+    transient_local: true
+    warn_rate: 0.0
+    error_rate: 0.0
+    timeout: 0.0
+
+- module: localization
+  mode: [online, planning_simulation]
+  type: autonomous
+  args:
+    node_name_suffix: initialpose3d
+    topic: /initialpose3d
+    topic_type: geometry_msgs/msg/PoseWithCovarianceStamped
+    best_effort: false
+    transient_local: false
+    warn_rate: 0.0
+    error_rate: 0.0
+    timeout: 0.0
+
+- module: localization
+  mode: [online]
+  type: autonomous
+  args:
+    node_name_suffix: pose_twist_fusion_filter_pose
+    topic: /localization/pose_twist_fusion_filter/pose
+    topic_type: geometry_msgs/msg/PoseStamped
+    best_effort: false
+    transient_local: false
+    warn_rate: 5.0
+    error_rate: 1.0
+    timeout: 1.0
+
+- module: sensing
+  mode: [online]
+  type: launch
+  args:
+    node_name_suffix: obstacle_segmentation_pointcloud
+    topic: /perception/obstacle_segmentation/pointcloud
+    topic_type: sensor_msgs/msg/PointCloud2
+    best_effort: true
+    transient_local: false
+    warn_rate: 5.0
+    error_rate: 1.0
+    timeout: 1.0
+
+- module: perception
+  mode: [online, planning_simulation]
+  type: autonomous
+  args:
+    node_name_suffix: object_recognition_objects
+    topic: /perception/object_recognition/objects
+    topic_type: autoware_auto_perception_msgs/msg/PredictedObjects
+    best_effort: false
+    transient_local: false
+    warn_rate: 5.0
+    error_rate: 1.0
+    timeout: 1.0
+
```

Comment:
Can traffic light recognition results be added?

Reply:
What is the topic name and expected rate?

Reply:
In planning simulation, the topic was not published periodically, so I will merge it once. Please change the config if necessary.